### ISSUE TITLE
The app is just an electron wrapper that makes all calls to pocketnet.app domain.

### ISSUE THREAD

**Issue by sexylust** [pocketnetteam/pocketnet.gui#58]
If the domain is taken down, the app will go down too.

You claim that it is resilient but apparently it is not.

**Comment by pocketnetteam** [comment]
Go to setting in the electron app and go to system. You can just actually switch the proxy. Soon it will be done automatically.
It is best to phrase these things as questions if you do not understand how it works.

**Comment by sexylust** [comment]
> Go to setting in the electron app and go to system. You can just actually switch the proxy. Soon it will be done automatically.
> It is best to phrase these things as questions if you do not understand how it works.

Valid point - I don't understand how it works yet. But I'm willing to help and contribute. That's why I started all this to begin with.

Regarding proxy and settings.
I assume you mean Settings when the user is logged in already. Yes, it is there. But if the user is not logged in, the Settings is not accessible and thus if domain is down, the app is not usable as the user won't be able to register.

You know how fast apps get cancelled these days and the domain can be taken down within a day, like Parler for example. So I consider it is quite a serious attack vector.

**Comment by pocketnetteam** [comment]
Alright, thank you for a productive discussion. Your offer help is appreciated. Please let us know how you can contribute (your specialties). User will definitely be able download the app and register even if the website is down. Soon (next release), we will have automatic proxy switching the app. So if the app gets known enough and there enough nodes (and there are already close to 100), the question will be in distributing the executable if the website is down. Could assume that github would be taken away too. We are working on those contingencies. But for now, we need a lot of help in finishing basic functionality, since this is the most ambitious and difficult project of this kind. The upside is that if completed, it will be the most censorship resistant by far. Let us know how you can help.

**Comment by sexylust** [comment]
Great! I'm quite good with React, javascript, node, java. Pretty much fullstack. I've just started exploring the GUI code and launched electron locally.
To be honest, the code structure is a bit unclear yet so I'm still getting familiar how it works on my free time but I hope I can contribute with pull requests and features.

**Comment by pocketnetteam** [comment]
Yes, much of the code is classic JS, which has to be rewritten. But the framework we are going to is more Vue.js.
The p2p chat we are doing is Vue, do you want to take on some work in that area?  We have also integrated PeerTube and there is some work there.
Backed is all C++. Take your pick)

**Comment by sexylust** [comment]
Yeah I saw backend. C++ is not my strength :)

Sure, Vue is fine too.

**Comment by sexylust** [comment]
@pocketnetteam How can I help?
